testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueryFramework.java

```diff
@@ -667,6 +669,16 @@ private Optional<DistributedQueryRunner> tryGetDistributedQueryRunner()
         return Optional.empty();
     }
 
+    protected TestTable createTable(String namePrefix, @Language("SQL") String tableDefinition)
+    {
+        return createTable(namePrefix, tableDefinition, ImmutableList.of());
+    }
+
+    protected TestTable createTable(String namePrefix, @Language("SQL") String tableDefinition, List<String> rowsToInsert)
+    {
+        return new TestTable(getQueryRunner()::execute, namePrefix, tableDefinition, rowsToInsert);
+    }
```

Comment:
nit: one thing I liked about the old was it made clear which executor is used to create table - maybe the method name should be `createTableUsingTrino` or something to clarify this?

This is a not a big deal so feel free to ignore.

Reply:
createTableUsingTrino
newTrinoTable
createTable